src/server/set_family.cc

```diff
@@ -1152,7 +1151,7 @@ void SDiff(CmdArgList args, Transaction* tx, SinkReplyBuilder* builder, Connecti
 
   tx->ScheduleSingleHop(std::move(cb));
   ResultSetView rsv = DiffResultVec(result_set, src_shard);
-  SetReplies{cntx}.Send(rsv);
+  SetReplies{builder, bool(cntx->conn_state.script_info)}.Send(rsv);
```

Comment:
May be static_cast<bool> would be better?

Reply:
I am not aware of any reason of why static_cast is better, so it falls under subjective preferences, imho.
As such, I usually prefer more concise variants.